components/04-reference-pages/_datatables-preview.hbs

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+   <title>UTSA</title>
+   <meta charset="utf-8" />
+   <meta name="keywords" content="UTSA" />
+   <meta name="viewport" content="width=device-width, initial-scale=1" />
+   <meta name="description" content="College Design Language System - Academic Strategic Communications - UTSA" />
+   <link rel="preload" href="https://use.typekit.net" as="font" />
+   <link rel="dns-prefetch" href="https://use.typekit.net" as="font">
+   <link rel="stylesheet" href="https://use.typekit.net/fhh1zvd.css" />
+   <link media="all" rel="stylesheet" href="https://cdn.datatables.net/2.2.2/css/dataTables.dataTables.min.css" />
```

Comment:
yeah, let's bring this into the repo, but we can have a separate js file if we want to avoid pulling into the bigger college-dls js file we build for everything else

Reply:
I installed the npm package, but feel free to let me know if you have any edits.